src/controllers/invitation/createInvitation.controller.js

```diff
@@ -0,0 +1,73 @@
+import { Invitation, User, Organization, Project } from "../../config/db.js"
+
+const createInvitationController = async (req, res) => {
+  // ObjetiveId is the Organization or Project id associate with the Invitation
+  const { objectiveId, email, type, accessType } = req.body
+  if (!email || !type || !objectiveId) {
+    return res.status(400).send("You must send an email and the required data")
+  }
+  if (type == "Project" && !accessType) return res.status(400).send("You must send an accessType")
+
+  const { currentUser } = req
+  if (currentUser.email == email)
+    return res.status(400).send("You can't send invitations to yourself")
+
+  // User who received the invitation
+  const userObjective = await User.findOne({ where: { email: email } })
+
+  if (!userObjective) return res.status(400).send("We couldn't find the user :(")
+
+  // Check if there is already an invitation
+  let prevInvitation
+  if (type == "project")
```

Comment:
check this condition to match the condition in line 9 
o sino hacer esto:
```suggestion
  if (type.toLowerCase() == "project")
```